### DISCUSSION TITLE
GraphQL API v4 PullRequestFilters Input Object

### DISCUSSION THREAD

**Discussion by kongo555** [community/community#24323] [Apps, API and Webhooks]
<p>I’m trying to get all issues and pull request since date. For issues it is easy:</p>
<pre><code>query {
  viewer {
    name
    repository(name: "test") {
      issues(first:10, filterBy: { since: "2018-11-06T10:43:22Z" }) {
        edges {
          node {
            title
            updatedAt
          }
        }
      }
    }
  }
}
</code></pre>
<p>Unfortunately same thing can’t be done for PullRequest. It is a bit strange beacuse PullRequest is an Issue anyway. Is there any chance for PullRequestFilters? Or search query is the only way to get it?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/kongo555">@kongo555</a>,</p>
<p>Thank you for being here! Here is a query that would help:</p>
<pre><code>{
  search(query: "repo:atom/atom is:pr is:open created:&gt;2019-04-01", type: ISSUE, last: 100) {
    edges {
      node {
        ... on PullRequest {
          url
          title
          createdAt
        }
      }
    }
  }
}
</code></pre>
<p>You’ll need to use pagination to fetch further results, and I’d recommend taking a read through the official documentation for using cursors and pages here: <a href="https://graphql.org/learn/pagination/" rel="nofollow noopener">https://graphql.org/learn/pagination/</a>. </p>

**Comment by tiegz** [comment]
<aside class="quote group-github-employees" data-username="AndreaGriffiths11" data-post="2" data-topic="14454">
<div class="title">
<div class="quote-controls"></div>
 AndreaGriffiths11:</div>
<blockquote>
<p><code>search(query: "repo:atom/atom is:pr is:open created:&gt;2019-04-01", type: ISSUE, last: 100)</code></p>
</blockquote>
</aside>
<p>Hi andreagriffiths11, thanks for the workaround. Do you know if there are plans to add filterBy to pullRequests?</p>
